staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -2381,7 +2397,89 @@ func (i *IngressDescriber) describeBackend(ns string, backend *networkingv1beta1
 	return formatEndpoints(endpoints, sets.NewString(spName))
 }
 
-func (i *IngressDescriber) describeIngress(ing *networkingv1beta1.Ingress, describerSettings DescriberSettings) (string, error) {
+func (i *IngressDescriber) describeBackendV1(ns string, backend *networkingv1.IngressBackend) string {
+	endpoints, err := i.client.CoreV1().Endpoints(ns).Get(context.TODO(), backend.Service.Name, metav1.GetOptions{})
+	if err != nil {
+		return fmt.Sprintf("<error: %v>", err)
+	}
+	service, err := i.client.CoreV1().Services(ns).Get(context.TODO(), backend.Service.Name, metav1.GetOptions{})
+	if err != nil {
+		return fmt.Sprintf("<error: %v>", err)
+	}
+	spName := ""
+	for i := range service.Spec.Ports {
+		sp := &service.Spec.Ports[i]
+		if backend.Service.Port.Number != 0 {
+			sNum := int64(backend.Service.Port.Number)
+			sp.Name = strconv.FormatInt(sNum, 10)
```

Comment:
match the logic above where we check if backend.Service.Port.Number matches `sp.Port`, and if so, set `spName = sp.Name`